galaxy/Dockerfile

```diff
@@ -122,7 +122,7 @@ RUN mkdir $GALAXY_ROOT/tool_deps
 #RUN export GALAXY=$GALAXY_ROOT && sh ./cron/updateucsc.sh.sample
 
 ENV GALAXY_CONFIG_DATABASE_CONNECTION=postgresql://galaxy:galaxy@localhost:5432/galaxy?client_encoding=utf8 \
-GALAXY_CONFIG_TOOL_DEPENDENCY_DIR=./tool_deps \
+GALAXY_CONFIG_TOOL_DEPENDENCY_DIR=$GALAXY_ROOT/tool_deps \
 GALAXY_CONFIG_ADMIN_USERS=admin@galaxy.org \
```

Comment:
See my other PR: https://github.com/bgruening/docker-galaxy-stable/pull/215
Is this one ready to go? If so I will update the other one.